drivers/hud_spm_report/app/models/hud_spm_report/fy2024/episode.rb

```diff
@@ -282,24 +314,35 @@ def compute_episode(enrollments, included_project_types:, excluded_project_types
     private def filter_episode(calculated_bed_nights)
       return unless calculated_bed_nights.present?
 
+      # Sort by date to ensure chronological order
       calculated_bed_nights = calculated_bed_nights.sort_by(&:last)
+      # Keep bed nights that are:
+      # - on or after the lookback date
+      # - OR associated with an enrollment that started on or after the lookback date
+      calculated_bed_nights = calculated_bed_nights.select do |enrollment, _, bed_night_date|
```

Comment:
fixes issue with counting days in ph enrollments with start date before lookback stop.